test/typecheck/OWNERS

```diff
@@ -5,4 +5,5 @@ reviewers:
   - cblecker
 approvers:
   - cblecker
-  - rmmh
```

Comment:
cc @cblecker -- I'd like to swap in on this one rather than have n=1, I think I already have test/ via an alias and I'm already in reviewers / have sent a PR or two to this.

Reply:
there's also a related directory in third_party/ that should be updated (though understandable if it's another PR)
https://github.com/kubernetes/kubernetes/blob/master/third_party/go-srcimporter/OWNERS

Reply:
I'm okay with adding @BenTheElder , but I'm also okay with it being done in a follow up.

Reply:
filed https://github.com/kubernetes/kubernetes/pull/84020 to follow up :smile: